optimizer_wrapper.rb

```diff
@@ -365,14 +340,17 @@ def self.split_independent_vrp_by_skills(vrp)
         independent_skills[set_index] += b_set
       }
     }
+
     # Original skills are retrieved
-    independant_skill_sets = independent_skills.map{ |index_set|
-      index_set.collect{ |index| mission_skills[index] }
+    independant_skill_sets = independent_skills.map{ |set_indices|
+      set_indices.collect{ |index| mission_skills[index] }
```

Comment:
I don't think, this change makes it more clear. It was better before. 

`index_set` literally means a set which contains indexes - a "set of indices" - but `set_indices` does not work.

```suggestion
    independant_skill_sets = independent_skills.map{ |index_set|
      index_set.collect{ |index| mission_skills[index] }
```